TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/commands/Com_Vision.java

```diff
@@ -0,0 +1,18 @@
+package org.firstinspires.ftc.teamcode.subsystems.commands;
+
+import com.arcrobotics.ftclib.command.CommandBase;
+
+import org.firstinspires.ftc.teamcode.subsystems.VisionSystem;
+
+public class Com_Vision extends CommandBase {
```

Comment:
Whoops! You need to add an isFinished here for the sequential command group. Just override it and automatically return true. That way it only runs once. You should actually make this like an instant command and have the initialize method read the current stack height instead of the execute.